templates/tools/dockerfile/php_fpm_nginx.include

```diff
@@ -0,0 +1,17 @@
+# Install nginx
+RUN apt-get install -y nginx-extras --no-install-recommends
+
+# Set config
+RUN sed -i "s|worker_processes 4|worker_processes 1|g" /etc/nginx/nginx.conf
+RUN sed -i "s|#location ~ \\\.php\$ {| location ~ \\\.php\$ {include snippets\/fastcgi-php\.conf;fastcgi_pass 127\.0\.0\.1:9000;}|g" /etc/nginx/sites-available/default
+
+RUN cp /usr/local/etc/php-fpm.conf.default /usr/local/etc/php-fpm.conf
+RUN cp /usr/local/etc/php-fpm.d/www.conf.default /usr/local/etc/php-fpm.d/www.conf
+RUN sed -i "s|include=NONE\/etc|include=etc|g" /usr/local/etc/php-fpm.conf
+
+RUN echo "extension=grpc.so" >> /usr/local/etc/php.ini
+
+# Enable nginx
+RUN service nginx start
```

Comment:
sounds like starting nging shouldn't really be part of the base image. Run it when you actually run the tests?

Reply:
Done. Thank you for helping me review the PR. I moved it to where the test starts. I am going to regenerate the Dockerfile and test it again this morning.